templates/pkg/resource/sdk_update.go.tpl

```diff
@@ -25,7 +25,11 @@ func (rm *resourceManager) sdkUpdate(
 		return updated, err
 	}
 {{- end }}
+	{{- if .CRD.OmitUnchangedFieldsOnUpdate }}
+	input, err := rm.newUpdateRequestPayload(ctx, desired, delta)
+	{{- else }}
 	input, err := rm.newUpdateRequestPayload(ctx, desired)
+	{{- end }}
```

Comment:
You could probably just always add the `delta` parameter to the `newUpdateRequestPayload` method, regardless of whether `OmitUnchangedFieldsOnUpdate` is true...

Reply:
+1